pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -132,6 +132,11 @@ def test_get_data_multiple_symbols(self):
         web.get_data_yahoo(sl, '2012')
 
     @skip_on_exception(RemoteDataError)
+    def test_get_data_null_as_missing_data(self):
+        # just test that we succeed
```

Comment:
I was thinking about the second test but realized that in the future pandas versions (0.21.0 and up)  `read_csv` will return 'np.nan' and all previous versions will return 'null'. We can check the pandas version but it is not yet released version not sure if there is a chance of the version number change.  Also, the test will depend on Yahoo data handling and they recently started to use 'null' as a missing value and can change it in the future. We can mock Yahoo data or pandas  `read_csv` function and in the latter case, there will be no need to check pandas version.